### ISSUE TITLE
Update the views / dashboards with the new FHIR resources on BQ

### ISSUE THREAD

**Issue by AnnieMungai** [opensrp/fhircore#1273]
**Is your feature request related to a problem? Please describe.**
@muneneg The family resources have been pushed to the BQ ([see issue #2704](https://github.com/onaio/canopy/issues/2704)).
cc @dubdabasoduba @pld

- [ ] Task: Update the views/ dashboard with the new resources.

Linked to issue #1154

**Comment by f-odhiambo** [comment]
**Update  - June 2,2022**
1. Currently, there is no automated way to get records from FHIR into the db we use for the dashboard.
2. Someone has to manually refresh, which will alter the lastupdated field in the db;  all records will have the date of the manual refresh.
3. On the dashboards, this means that all records will show the last updated date an not the date they were actually created.
4. Proposed ways to counter this:
5. Update the pipeline to create a new column for when the data hits the db
6. Amend the app and include a tag for the date created, which will then be used moving forward.

**Next steps:**

1. Set-up a call with [@pld](https://onaio.slack.com/team/U03JETLL9) to make a decision on the best and most efficient way to handle this -within the week- [@Mwalimu](https://onaio.slack.com/team/UC2DNU0F3)
2. Follow-up on the progress of the sync bug that was preventing data on the single approach apk from getting to the server and give an update by EOD  [@Mwalimu](https://onaio.slack.com/team/UC2DNU0F3)


Related SDK ticket - https://github.com/google/android-fhir/issues/1420

**Comment by pld** [comment]
I don't think that SDK ticket is related to this issue, we need to find an appropriate place for this issue, this isn't the correct repo
